synapse/lib/layer.py

```diff
@@ -24,13 +23,22 @@ class Utf8er(collections.defaultdict):
     def __missing__(self, name):
         return name.encode('utf8')
 
+class LayerApi(s_cell.PassThroughApi):
+    allowed_methods = [
+        'getLiftRows', 'stor', 'commit', 'abort', 'getBuidProps',
+        'iterFormRows', 'iterPropRows', 'iterUnivRows', 'getOffset',
+        'setOffset', 'initdb', 'splicelistAppend', 'splices', 'stat'
```

Comment:
do we want to make `setOffset` an adminApi? or defer that?

Reply:
I'd like to defer that since the whole API is kind admin. Plus, we might remove the splice stuff from the layer altogether.